### PR TITLE
Rover stories V3

### PR DESCRIPTION
**Pull request by jabrilmahamud**

### CODE REVIEW

**Review by jabrilmahamud — COMMENTED** [review]
LGTM, I deleted config.js.

File: src/backend/apiRoutes.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import express from "express";
 import QuotationsController from "./controllers/QuotationsController.js";
+import PhotosController from "./controllers/PhotosController"
+const router = express.Router();
+
+router.use("/quotations", QuotationsController);
+router.use("/photos",PhotosController)
```
Comment (Kenneth-Siu) [inline src/backend/apiRoutes.js]: Why's this changed in a ticket about adding rover descriptions to the homepage frontend?

I see it's not intended to change anything - can you just put everything back to how it was? You can use this file to see what you've changed to make sure everything goes back the way it was.

File: src/backend/controllers/PhotosController.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import express from "express";
```
Comment (Kenneth-Siu) [inline src/backend/controllers/PhotosController.js]: Again, is this file supposed to be here?

**Review by Kenneth-Siu — CHANGES_REQUESTED** [review]
There are some mysterious files being added/changed that shouldn't be, but the actual main part of the ticket looks really good!

**Pull request closed without merging**